### ISSUE TITLE
failed to compile `racer v2.0.2`, intermediate artifacts can be found at `/tmp/cargo-install.v3v5uYgW11Sk`

### ISSUE THREAD

**Issue by htory327** [phildawes/racer#670]
[uacfw@uacfw ~/Desktop/Rust/projects]$ sudo cargo install racer
Password:
    Updating registry `https://github.com/rust-lang/crates.io-index`
   Compiling ansi_term v0.9.0
   Compiling bitflags v0.7.0
   Compiling strsim v0.5.2
   Compiling typed-arena v1.2.0
   Compiling unicode-width v0.1.4
   Compiling rustc-serialize v0.3.22
   Compiling log v0.3.6
   Compiling utf8-ranges v0.1.3
   Compiling regex-syntax v0.3.9
   Compiling libc v0.2.18
   Compiling winapi v0.2.8
   Compiling term_size v0.2.1
   Compiling memchr v0.1.11
   Compiling aho-corasick v0.5.3
   Compiling unicode-xid v0.0.3
   Compiling unicode-segmentation v0.1.3
   Compiling vec_map v0.6.0
   Compiling winapi-build v0.1.1
   Compiling clap v2.19.2
   Compiling kernel32-sys v0.2.2
error: linking with `cc` failed: exit code: 69
  |
  = note: "cc" "-m64" "-L" "/Users/uacftw/.rustup/toolchains/stable-x86_64-apple-darwin/lib/rustlib/x86_64-apple-darwin/lib" "/tmp/cargo-install.v3v5uYgW11Sk/release/build/kernel32-sys-d6afa5bd3d7cfaef/build_script_build.0.o" "-o" "/tmp/cargo-install.v3v5uYgW11Sk/release/build/kernel32-sys-d6afa5bd3d7cfaef/build_script_build" "-Wl,-dead_strip" "-nodefaultlibs" "-L" "/tmp/cargo-install.v3v5uYgW11Sk/release/deps" "-L" "/Users/uacftw/.rustup/toolchains/stable-x86_64-apple-darwin/lib/rustlib/x86_64-apple-darwin/lib" "/private/tmp/cargo-install.v3v5uYgW11Sk/release/deps/libbuild-493a7b0628804707.rlib" "/Users/uacftw/.rustup/toolchains/stable-x86_64-apple-darwin/lib/rustlib/x86_64-apple-darwin/lib/libstd-f5a209a9.rlib" "/Users/uacftw/.rustup/toolchains/stable-x86_64-apple-darwin/lib/rustlib/x86_64-apple-darwin/lib/libpanic_unwind-f5a209a9.rlib" "/Users/uacftw/.rustup/toolchains/stable-x86_64-apple-darwin/lib/rustlib/x86_64-apple-darwin/lib/libunwind-f5a209a9.rlib" "/Users/uacftw/.rustup/toolchains/stable-x86_64-apple-darwin/lib/rustlib/x86_64-apple-darwin/lib/librand-f5a209a9.rlib" "/Users/uacftw/.rustup/toolchains/stable-x86_64-apple-darwin/lib/rustlib/x86_64-apple-darwin/lib/libcollections-f5a209a9.rlib" "/Users/uacftw/.rustup/toolchains/stable-x86_64-apple-darwin/lib/rustlib/x86_64-apple-darwin/lib/librustc_unicode-f5a209a9.rlib" "/Users/uacftw/.rustup/toolchains/stable-x86_64-apple-darwin/lib/rustlib/x86_64-apple-darwin/lib/liballoc-f5a209a9.rlib" "/Users/uacftw/.rustup/toolchains/stable-x86_64-apple-darwin/lib/rustlib/x86_64-apple-darwin/lib/liballoc_jemalloc-f5a209a9.rlib" "/Users/uacftw/.rustup/toolchains/stable-x86_64-apple-darwin/lib/rustlib/x86_64-apple-darwin/lib/liblibc-f5a209a9.rlib" "/Users/uacftw/.rustup/toolchains/stable-x86_64-apple-darwin/lib/rustlib/x86_64-apple-darwin/lib/libcore-f5a209a9.rlib" "/Users/uacftw/.rustup/toolchains/stable-x86_64-apple-darwin/lib/rustlib/x86_64-apple-darwin/lib/libcompiler_builtins-f5a209a9.rlib" "-l" "System" "-l" "pthread" "-l" "c" "-l" "m"
  = note: 
You have not agreed to the Xcode license agreements, please run 'xcodebuild -license' (for user-level acceptance) or 'sudo xcodebuild -license' (for system-wide acceptance) from within a Terminal window to review and agree to the Xcode license agreements.

error: aborting due to previous error

Build failed, waiting for other jobs to finish...
error: failed to compile `racer v2.0.2`, intermediate artifacts can be found at `/tmp/cargo-install.v3v5uYgW11Sk`

Caused by:
  Could not compile `kernel32-sys`.

To learn more, run the command again with --verbose.

Used sudo because previous error told me to try _with_ sudo

**Comment by pmarcelll** [comment]
Maybe [this](http://stackoverflow.com/questions/27907566/failed-with-exit-status-69) will help.

**Comment by htory327** [comment]
Thank you, that fixed it. I had to agree to xCode License that I didn't know I had to

TL;DR to solve this you have to just write `sudo xcrun cc` and you're game.
